helm/oauth2-proxy/values.yaml

```diff
@@ -227,6 +227,17 @@ podSecurityContext: {}
 # whether to use http or https
 httpScheme: http
 
+initContainers:
+  # if the redis sub-chart is enabled, wait for it to be ready
+  # before starting the proxy
+  # creates a role binding to get, list, watch, the redis master pod
+  # if service account is enabled
+  waitForRedis:
+    enabled: true
```

Comment:
Actually, we can't enable this by default since Redis itself is disabled AND default storage type is `cookie`. I know this makes a lot of changes in order to enable this, thus maybe you could edit the `redis-standalone-values.yaml` to show this, then reference this file from the README as a guide to enable redis.

Reply:
@bluebrown I would say so we can keep it `false` by default!
If you implement the documentation as requested by Cédric I would say that we are ready to move the PR forward. 🚀

Reply:
if redis is not enabled, the init container is not deployed, even if this is set to true. you can see this condition in the deployment.